Create documentation for EAPTS adaptor

mercy-ship-it opened the pull request:
## Short Description

This PR adds comprehensive integration documentation for the EAPTS (Electronic Asset and Pharmaceutical Tracking System) adaptor

## Details

- **App Overview**
- **Data Model**
- **Integration Use Cases**

## AI Usage

Please disclose how you've used AI in this work (it's cool, we just want to
know!):

- [ ] Code generation (copilot but not intellisense)
- [ ] Learning or fact checking
- [x] Strategy / design
- [ ] Optimisation / refactoring
- [ ] Translation / spellchecking / doc gen
- [ ] Other
- [ ] I have not used AI

You can read more details in our
[Responsible AI Policy](https://www.openfn.org/ai#pull-request-templates)

mercy-ship-it committed:
Create documentation for EAPTS adaptor

Add documentation for EAPTS adaptor including overview, functions, and interfaces.

hunterachieng reviewed (commented):
@mercy-ship-it  If we look at the other docs, the format is different from the one you have written

mercy-ship-it committed:
Revise EAPTS adaptor documentation

Updated documentation for the EAPTS adaptor, including an overview, integration options, and functions.

mercy-ship-it committed:
Updated eapts.md

mercy-ship-it committed:
Update eapts.md

mercy-ship-it committed:
Updated eapts.md

mercy-ship-it reviewed (commented):
@hunterachieng I have updated the Docs

josephjclark commented:
Hi @hunterachieng should we merge this? Or close it? Or ask someone else?

hunterachieng commented:
@josephjclark  Yes we should merge
